src/app/tasks/tasks.component.html

```diff
@@ -15,6 +15,7 @@
     <mat-checkbox [checked]="task.completed"></mat-checkbox>
     <p matLine>{{task.title}}</p>
     <p matLine><ng-container i18n>Deadline:</ng-container> {{task.deadline | date}}</p>
+    <p matLine *ngIf="task.completed"><ng-container i18n>Completed Time: </ng-container> {{task.completedTime | date:'MMM d, y, h:mm a'}}</p>
```

Comment:
Let's remove this, but also for future reference please use generic date pipes (the default, or `longDate`, `medium`, etc) so the exact format is set by the locale.